### PR TITLE
uac

### PR DESCRIPTION
**Pull request by tccontre**
## UAC Bypassed Detections

1. SilentCleanup UAC Bypass
2. Sdclt UAC Bypass
3. WSReset UAC Bypass
4. NET Profiler UAC bypass
5. UAC Bypass MMC Load Unsigned Dll

## trickbot detection
1. Mshta spawning Rundll32 OR Regsvr32 Process
2. Office Product Spawn CMD Process

### CODE REVIEW

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

UAC Bypass MMC Load Unsigned Dll :white_check_mark:
- diskUsage: 163840
- runDuration: 1.112s
- scanCount: 8

File: detections/endpoint/sdclt_uac_bypass.yml [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  confidence: 90
+  # (impact * confidence)/100
+  risk_score: 63
+  context:
+  - Source:Endpoint
+  - Stage:Persistence,
+  - Stage:Privilege Escalation
+  - Stage:Defense Evasion
+  - Scope:Incoming
+  message: Suspicious modification of registry $Registry.registry_path$ with possible payload path $Registry.registry_value_name$ in $Registry.dest$
```
Comment (patel-bhavin) [inline detections/endpoint/sdclt_uac_bypass.yml]: the message should contain only the field name `registry_path` and not `Registry.registry_path`

Comment (tccontre) [reply detections/endpoint/sdclt_uac_bypass.yml]: thanks. will fix this one now

**Comment by P4T12ICK** [comment]
This PR was created by Automated Detection Testing Service :robot:

Tested detections:

UAC Bypass MMC Load Unsigned Dll :white_check_mark:
- diskUsage: 172032
- runDuration: 1.807s
- scanCount: 8

**Review by P4T12ICK — APPROVED** [review]
Looks good to me

**Pull request merged**